release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -3422,4 +3422,28 @@ $ oc adm release info 4.11.34 --pullspecs
 [id="ocp-4-11-34-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-11-35"]
+=== RHBA-2023:1650 - {product-title} 4.11.35 bug fix
+
+Issued: 2023-04-11
+
+{product-title} release 4.11.35 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:1650[RHBA-2023:1650] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:1649[RHBA-2023:1649] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.11.35 --pullspecs
+----
+
+[id="ocp-4-11-35-bug-fixes"]
+==== Bug fixes
+* Previously, `machine-api-provider-openstack` had to be restarted to pick up new cloud credentials in the event of a rotation of the OpenStack `clouds.yaml`. As a result, the ability of a `MachineSet` to scale to zero could be affected. With this change, cloud credentials are no longer cached and `machine-api-provider-openstack` will read the corresponding secret fresh when it's needed. (link:https://issues.redhat.com/browse/OCPBUGS-10954[*OCPBUGS-10954*])
```

Comment:
```suggestion
* Previously, if the OpenStack `clouds.yaml` file was rotated, you needed to restart `machine-api-provider-openstack` to pick up  new cloud credentials. As a result, the ability of a `MachineSet` to scale to zero could be affected. With this change, cloud credentials are no longer cached and `machine-api-provider-openstack` reads the corresponding secret fresh when it is needed. (link:https://issues.redhat.com/browse/OCPBUGS-10954[*OCPBUGS-10954*])
```
I'm not sure what "...reads the corresponding secret fresh..." means. Is a "secret fresh" a thing? Or is "fresh" modifying another word? I'm wondering if we can remove "fresh" altogether.

Reply:
I think you are right that we can leave fresh out. Thank you!